src/algebra/group/units.lean

```diff
@@ -116,8 +116,10 @@ lemma copy_eq (u : αˣ) (val hv inv hi) :
   u.copy val hv inv hi = u :=
 ext hv
 
+-- Needs priority `< 100` to make division of units in `[comm_ring α]` work properly.
 /-- Units of a monoid form a group. -/
-@[to_additive "Additive units of an additive monoid form an additive group."] instance : group αˣ :=
+@[to_additive "Additive units of an additive monoid form an additive group.", priority 70]
```

Comment:
If you split this into separate `has_mul`, `has_one`, and `group` instances, is it better behaved?